X16 Reference - 10 - VERA FX Reference.md

```diff
@@ -37,7 +37,7 @@ VERA is mapped as 32 8-bit registers in the memory space of the Commander X16, s
 		<td>$9F25</td>
 		<td>CTRL</td>
 		<td colspan="1" align="center">Reset</td>
-		<td colspan="6" align="center"><b>DCSEL</b><br><img src="images/redunderline.png"></td>
+		<td colspan="6" align="center"><b>DCSEL</b><br><img src="/images/redunderline.png"></td>
```

Comment:
are you sure we want absolute paths here? I think we should always stick to relative ones.

Reply:
Fixed via markdown setting (though did convert some of those to markdown image tags)